### DISCUSSION TITLE
[FEATURE] Filter validation results on single expectation

### DISCUSSION THREAD

**Discussion by MatteoFelici** [great-expectations/great_expectations#5081] [Ideas]
In data docs, I'd like to search for all validation failures on a single expectation. For now, you can filter on failure/success, but I cannot, for example, get all validation results where the expectation on feature `email` failed.

**Comment by NathanFarmer** [comment]
Hi @MatteoFelici! Thanks for opening this issue. We will discuss this internally and respond soon. In the meantime if you would like to contribute this feature, we would be happy to offer you and guidance needed.

**Comment by NathanFarmer** [comment]
Hi @MatteoFelici. This feature request has been moved into our engineering backlog. We will update you when we begin the work, but it is not up for development this planning cycle. In the meantime, we welcome a contribution on this feature and would appreciate an update if you or someone starts work on it.
